JUnitParamsAndMockito_task5/src/main/java/org/Marta/SeasonChecker.java

```diff
@@ -0,0 +1,34 @@
+package org.Marta;
+
+import java.time.Month;
+
+public class SeasonChecker {
+    public static String getSeason(Month month) {
+        if (month == null) {
+            return "";
+        }
+        String season = "";
```

Comment:
Można to zrobić ciut ladniej pomijając tworzenie tej zmiennej. Zamiast:
```
 season = Season.WINTER.toString();
```
zrobić:

```
 return Season.WINTER.toString();
```